backend/de.metas.contracts/src/main/java/de/metas/contracts/bpartner/process/C_BPartner_MoveToAnotherOrg_ProcessHelper.java

```diff
@@ -120,6 +120,8 @@ public void onParameterChanged(final String parameterName)
 			isShowMembershipParameter = orgChangePartnerComposite.hasMembershipSubscriptions()
 					&& service.hasAnyMembershipProduct(p_orgTargetId);
 
+			p_groupTemplateId = orgChangePartnerComposite.getGroupCategoryId() == null ? null : orgChangePartnerComposite.getGroupCategoryId();
```

Comment:
hmmm, why not assigning it directly?
i.e.
```java
p_groupTemplateId = orgChangePartnerComposite.getGroupCategoryId();
```

Reply:
fixed